src/runtime/Context.h

```diff
@@ -363,6 +363,7 @@ class Context : public gc {
     ObjectStructure* m_defaultStructureForRegExpObject;
     ObjectStructure* m_defaultStructureForMappedArgumentsObject;
     ObjectStructure* m_defaultStructureForUnmappedArgumentsObject;
+    ObjectStructure* m_defaultStructureForErrorObject;
```

Comment:
Is this default structure actually used?

Reply:
Opps!!!